source/future.tex

```diff
@@ -1468,8 +1468,7 @@
 only if the type \tcode{T} is any of the following:
 \begin{itemize}
 \item a function type or a pointer to function type taking one argument of type \tcode{T1}
-% FIXME: Should this be R T0::f() ?
-\item a pointer to member function \tcode{R T0::f} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
+\item a pointer to member function \tcode{R T0::f()} \cv{} (where \cv{} represents the member function's cv-qualifiers); the type \tcode{T1} is \cv{} \tcode{T0*}
```

Comment:
`tr1::reference_wrapper` says:

> — a pointer to member function type with cv-qualifier _cv_ and no arguments; the type `T1` is _cv_ `T*` and `R` is the return type of the pointer to member function

and then for the `R T0::f(T2)` case:

> — a pointer to member function with cv-qualifier _cv_ and taking one argument of type `T2`; the type `T1` is _cv_ `T*` and `R` is the return type of the pointer to member function

So at some point we rephrased that and mangled it.